digitaltwin/service/src/Microsoft.Azure.Devices.DigitalTwin.Service.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <!-- delay sign the assembly for Release build -->
```

Comment:
Same as above - Why is this commented out?

Reply:
I've uncommented it now, good catch